.packit.yaml

```diff
@@ -0,0 +1,52 @@
+specfile_path: packaging/leapp.spec
+
+upstream_package_name: leapp
+downstream_package_name: leapp
+upstream_project_url: https://github.com/oamg/leapp
+
+jobs:
+- job: copr_build
+  trigger: pull_request
+  metadata:
+    owner: "@oamg"
+    project: leapp
+    targets:
+    - epel-7-ppc64le
+    - epel-7-x86_64
+    - epel-8-x86_64
+    - fedora-33-aarch64
+    - fedora-33-x86_64
+    - fedora-34-aarch64
+    - fedora-34-x86_64
+    - fedora-rawhide-aarch64
+    - fedora-rawhide-x86_64
+  actions:
+    post-upstream-clone:
+    # builds from PRs should have lower NVR than those from master branch
+    - sed -i "s/1%{?dist}/0%{?dist}/g" packaging/leapp.spec
```

Comment:
In Convert2RHEL, @TomasTomecek used `rpmdev-bumpspec`:
https://github.com/oamg/convert2rhel/blob/v0.22/.packit.yaml#L34

I don't know what's better so take this just as FYI.

Reply:
Actually I used c2r's packit config as a starting point but wanted to keep the naming schema as much similiar as possible to the already existing one, that's why I'm using `sed` to set the release number to `100`.

Reply:
it's really just a matter of preference: rpmdev-bumpspec actually parses the spec file and does some magic which can lead to inconsistent results while sed is very explicit - just please make sure to get the regex right so it doesn't change more things than you'd want it to

Reply:
The exact same regex is used in [Makefile](https://github.com/oamg/leapp/blob/408d99628ad4b1c35542f7d2430cf145281c34f5/Makefile#L91) for quite some time so I believe it should be reasonably safe.